spec/Section 3 -- Type System.md

```diff
@@ -248,6 +248,29 @@ a given GraphQL server expects. Any other input value, including float input
 values (such as `4.0`), must raise a query error indicating an incorrect type.
 
 
+#### Any
+
+The Any scalar type represents any value that can be represented by underlying
+serialization protocol (including lists and maps). It is intended to be an opt-out type in case when the
+exact type of possible data is not known in advance or multiple types should be
```

Comment:
I think suggesting supporting multiple types with `Any` isn't quite correct. Maybe a better way to phrase this is in terms of input types - where an input wishes to accept any kind of data.